src/main/java/org/opensearch/knn/index/util/IndexInputWithBuffer.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.util;
```

Comment:
I won't put this under util package. `org.opensearch.knn.index.store` should be fine.

Reply:
Sounds good! Will do in the next rev